app/src/main/java/openfoodfacts/github/scrachx/openfood/features/product/view/summary/SummaryProductFragment.kt

```diff
@@ -462,6 +474,44 @@ class SummaryProductFragment : BaseFragment(), ISummaryProductPresenter.View {
         }
     }
 
+    private fun showListChips() {
+
+        // remove the existing childviews on chip group if any
+        binding.listChips.removeAllViews()
+
+        val listDao: YourListedProductDao = OFFApplication.daoSession.yourListedProductDao
+
+        //Query to check if the product belons in any of thr user's list
+        val queryResult : List<YourListedProduct> = listDao.queryBuilder()
```

Comment:
This operation is on the main thread, we do not want it.

Reply:
Changed the thread. Will push the change after layout is finalized.